pepper-apis/src/main/resources/org/broadinstitute/ddp/db/dao/ActivityInstanceSql.sql.stg

```diff
@@ -39,7 +39,14 @@ select ai.activity_instance_id   as a_instance_id,
        nt.numeric_type_code as na_numeric_type,
        na.int_value         as na_int_value,
        po.picklist_option_stable_id  as pa_option_stable_id,
-       ppo.picklist_option_stable_id as pa_parent_option_stable_id,
+       (select ppo.picklist_option_stable_id from picklist_nested_option as npo join picklist_option as ppo
```

Comment:
Should we make it a little more consistent and make the alias `pno` for `picklist_nested_option`?